readme.md

```diff
@@ -36,9 +30,9 @@ Today, we will get practice writing the underlying code responsible for adding b
 
 ### jQuery and Vanilla Javascript
 
-Note that we are using jQuery for this lesson. jQuery is a **library** written in javascript that allows us to do more with less code. There may be situations where we'd want to use vanilla javascript over jQuery to write our code. This might include reverse compatibility (older versions of IE). Check out [this](http://youmightnotneedjquery.com/) website for side-by-side comparisons of jQuery and vanilla javascript.
+Note that we are using jQuery for this lesson. jQuery is a **library** written in javascript that allows us to do more with less code. Check out [this](http://youmightnotneedjquery.com/) website for side-by-side comparisons of jQuery and vanilla javascript.
 
```

Comment:
Nah, I'm just going to leave it in there for the link really, in case people want to talk about **why** we're using it. Not going to dwell on this at all though.